src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -60,7 +66,16 @@ public ApiSettings(NodeSettings nodeSettings)
 
             TextFileConfiguration config = nodeSettings.ConfigReader;
 
-            string apiHost = config.GetOrDefault("apiuri", DefaultApiHost, this.logger);
+            this.UseHttps = config.GetOrDefault("usehttps", false);
+            this.HttpsCertificateFilePath = config.GetOrDefault("certificatefilepath", (string)null);
+            if(this.UseHttps && string.IsNullOrWhiteSpace(this.HttpsCertificateFilePath))
+                throw new ConfigurationException("The path to a certificate needs to be provided when using https. Please use the argument 'certificatefilepath' to provide it.");
+
+            var defaultApiHost = this.UseHttps 
+                                     ? DefaultApiHost.Replace(@"http://", @"https://") 
```

Comment:
Just use normal indentation of four spaces

Reply:
ok...